### PR TITLE
Feature/progress component

### PR DESCRIPTION
**Pull request by DF981339**
Progress Component Done, including testing.

### CODE REVIEW

File: src/components/Progress/Progress.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,346 @@
...
+
+  return <Progress pgValue={progress} />;
+};
+
+export const DiffTypeProgress = () => {
+  const progress = useSimulateProgress();
+
+  return (
+    <div>
+      <h1>Linear</h1>
```
Comment (Patrick-FRE) [inline src/components/Progress/Progress.stories.tsx:21]: let ppl know the default one is Linear

File: src/components/Progress/Progress.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,346 @@
...
+
+      <h3>xs</h3>
+      <Progress pgSize="xs" pgValue={progress} />
+      <br />
+
+      <h3>sm</h3>
+      <Progress pgSize="sm" pgValue={progress} />
+      <br />
+
+      <h3>default (no need to add props)</h3>
```
Comment (Patrick-FRE) [inline src/components/Progress/Progress.stories.tsx:50]: make the default size more clear to user

File: src/components/Progress/Progress.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,346 @@
...
+export const ShowPercentage = () => {
+  const progress = useSimulateProgress();
+
+  return (
+    <div>
+      <h1>Linear</h1>
+      <br />
+
+      <h3>xs</h3>
+      <Progress showPercentage pgSize="xs" pgValue={progress} />
```
Comment (Patrick-FRE) [inline src/components/Progress/Progress.stories.tsx:126]: let user know what is the default value for show percentage

File: src/components/Progress/_Progress.scss [hunk 4]
```diff
@@ -0,0 +1,239 @@
...
+// linear text
+.pg-linear-text {
+  margin-left: 10px;
+  width: 5%;
+}
+
+// linear size
+.pg-linear-xs {
+  .pg-linear-bar {
+    height: 3px;
```
Comment (Patrick-FRE) [inline src/components/Progress/_Progress.scss:25]: create variable for the height

**Review by Patrick-FRE — APPROVED** [review]

**Review by Patrick-FRE — COMMENTED** [review]
good to merge

File: src/components/Progress/Progress.stories.tsx (hunk 2 above)
Comment (DF981339) [reply src/components/Progress/Progress.stories.tsx]: done

File: src/components/Progress/_Progress.scss (hunk 4 above)
Comment (DF981339) [reply src/components/Progress/_Progress.scss]: done

**Pull request merged**